invenio/modules/tags/models.py

```diff
@@ -257,7 +257,8 @@ def __init__(self, bibrec=None, **kwargs):
         """TODO."""
         super(WtgTAGRecord, self).__init__(**kwargs)
 
-        self.bibrec = bibrec
+        if bibrec is not None:
```

Comment:
It's defined as a SQLAlchemy relationship.
Do you have any idea why is implemented the `__init__`?
Because, with SQLAlchemy 1.0.0, tests are broken by init class and setting `self.bibrec = None` here.